src/fdosecrets/dbus/DBusObject.h

```diff
@@ -56,7 +56,7 @@ namespace FdoSecrets
             return m_objectPath;
         }
 
-        DBusMgr& dbus() const
+        QSharedPointer<DBusMgr> dbus() const
```

Comment:
This could return a const reference, to avoid unnecessary copy.

Reply:
Good point, will make that change

Reply:
I mean
`const QSharedPointer<DBusMgr>& dbus()`

This way those needed can still make a copy but temporary access like `dbus()->registerObject` saves a copy

Reply:
all done, waiting for CI to finish and then squash-rebase.